src/main/java/core/basesyntax/garage/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.garage;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
The message printed should be more informative. It should indicate what specific type of 'Machine' is working. Consider including the model or type of the Truck in the message.